torch/distributed/_symmetric_memory/__init__.py

```diff
@@ -22,11 +64,30 @@
 
 
 def enable_symm_mem_for_group(group_name: str) -> None:
-    """
-    Enables symmetric memory for a process group.
+    r"""enable_symm_mem_for_group(group_name) -> None
+
+    Enables symmetric memory for a process group. This function sets up the
+    necessary infrastructure to allow symmetric memory operations within the
+    specified process group.
+
+    This function is idempotent - calling it multiple times for the same group
+    has no additional effect. The function is automatically called by
+    :func:`get_symm_mem_workspace` and :func:`rendezvous`, so users typically
+    do not need to call it directly.
 
     Args:
-        group_name (str): the name of the process group.
+        group_name (str): The name of the process group for which to enable
+            symmetric memory. This must be a valid process group that has been
+            previously created via :func:`torch.distributed.new_group` or the
+            default ``"0"`` for the world group.
```

Comment:
This looks sus

Reply:
That looks valid :) "0" is the group name for the global group.